### PR TITLE
DEVDOCS-4338: [edit] Sites API typos

### PR DESCRIPTION
**Pull request by stanzikratelbc**
# [DEVDOCS-4338](https://bigcommercecloud.atlassian.net/browse/DEVDOCS-4338)

## What changed?
* change TSL to TLS
* fix typo
* make minor updates for punctuation consistency

### CODE REVIEW

File: reference/sites.v3.yml [hunk 1]
```diff
@@ -859,9 +859,9 @@ components:
             The route template that will be used to generate the URL for the requested resource.
 
             Supports several tokens:
-            - `{id}` The **ID** of the requested item
-            - `{slug}` The **slug** for the requested item (if available). Note: the `slug` value may contain `/` slash
-            - `{language}` The **language** string that the client is using
+            - `{id}` The **ID** of the requested item.
+            - `{slug}` The **slug** for the requested item (if available). Note: the `slug` value may contain `/` slash,
```
Comment (bc-andreadao) [inline reference/sites.v3.yml:863]:
```suggestion
            - `{slug}` The **slug** for the requested item (if available). Note: the `slug` value may contain `/` slash.
```

File: reference/sites.v3.yml [hunk 2]
```diff
@@ -307,7 +307,7 @@ paths:
             type: integer
         - name: type
           in: query
-          description: Filter routes by a given resource type
+          description: Filter routes by a specified resource type
```
Comment (bc-andreadao) [inline reference/sites.v3.yml:310]:
```suggestion
          description: Filter routes by a specified resource type.
```

**Comment by slsriehl** [comment]
@stanzikratelbc engage with @bc-andreadao's notes, then re-request approval from her.

**Comment by stanzikratelbc** [comment]
@bc-andreadao thank you for your suggestions. I have committed them. Can you please re-review? Thanks!

**Review by bc-andreadao — APPROVED** [review]

**Review by markcmurphy — APPROVED** [review]

**Pull request merged**